vignettes/manual/Hector-Wider-World.Rmd

```diff
@@ -100,6 +100,6 @@ The HectorUI interface can be found [here](https://jgcri.shinyapps.io/HectorUI/)
 
 The HectorUI paper can be found [here](https://joss.theoj.org/papers/10.21105/joss.02782)  
 
-> Evanoff J.E., C.R. Vernon, S.T. Waldhoff, A.C. Snyder, and C.A. Hartin. 2020. "hectorui: A web-based interactive scenario builder and visualization application for the Hector climate model." The Journal of Open Source Software 5, no. 56:Article No. 2782. PNNL-SA-156861. https://doi.org/10.21105/joss.02782
+> Evanoff J.E., Vernon, C.R., Waldhoff, S.T., Snyder, A. C., and Hartin, C.A. (2020). "hectorui: A web-based interactive scenario builder and visualization application for the Hector climate model." The Journal of Open Source Software 5, no. 56:Article No. 2782. PNNL-SA-156861. https://doi.org/10.21105/joss.02782
 
-A YouTube overview of HectorUI by Stephanie Pennington can be found [here](https://www.youtube.com/watch?v=fBHXS7pjZcI).
```

Comment:
Do we include @dawnlwoodard 's [permafrost paper](https://gmd.copernicus.org/articles/14/4751/2021/gmd-14-4751-2021-discussion.html) here? If not, we should.

Reply:
It's the first paper in the journal articles section

Reply:
Doh, sorry. k.